migrate/20240125_add_minio_crts.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:minio_cluster) do
+      add_column :root_cert_1, :text, collate: '"C"'
+      add_column :root_cert_key_1, :text, collate: '"C"'
+      add_column :root_cert_2, :text, collate: '"C"'
+      add_column :root_cert_key_2, :text, collate: '"C"'
+      add_column :certificate_last_checked_at, :timestamptz, null: false, default: Sequel.lit("now()")
```

Comment:
This seems to be not used at the moment. I guess you added this for certificate rotation, right?